src/style.css

```diff
@@ -1 +1,76 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
```

Comment:
Ensure that the font URL `fonts/Avenir-Book.ttf` is correct and accessible. Consider using a relative path consistently, such as `./fonts/Avenir-Book.ttf`, to avoid potential issues.